src/container/srv_layout.h

```diff
@@ -45,9 +45,17 @@
  * ds_cont_prop_version stores the version of the whole layout.
  */
 extern d_iov_t ds_cont_prop_version;		/* uint32_t */
+extern d_iov_t ds_cont_prop_cuuids;		/* container UUIDs KVS */
 extern d_iov_t ds_cont_prop_conts;		/* container KVS */
 extern d_iov_t ds_cont_prop_cont_handles;	/* container handle KVS */
 
+/*
+ * Container UUIDs KVS (RDB_KVS_GENERIC)
+ *
+ * This maps container labels to container UUID.
+ * Used to get UUID key for lookup in container KVS.
```

Comment:
[Minor] Should specify the type of each key and each value:

    This maps container labels (string, without '\0') to container UUIDs (uuid_t).

Reply:
Done.